packages/sdk-js/src/types/growthbook.ts

```diff
@@ -339,11 +360,13 @@ export type LoadFeaturesOptions = {
   autoRefresh?: boolean;
   timeout?: number;
   skipCache?: boolean;
+  useStoredPayload?: boolean;
 };
 
 export type RefreshFeaturesOptions = {
   timeout?: number;
   skipCache?: boolean;
+  useStoredPayload?: boolean;
```

Comment:
Isn't the skipCache option good enough by itself? If true, you would skip both the stored payload and the cache. I can't think of a good reason why you would want to skip the stored payload, but still use the cache.

Reply:
pending considerations about init()